index.html

```diff
@@ -0,0 +1,185 @@
+<!doctype html>
```

Comment:
WTF is this file for?

Reply:
it was just bootstrap-code-helper file=D
already deleted